### DISCUSSION TITLE
Can't setting up the "Intercom" custom domain for the help center

### DISCUSSION THREAD

**Discussion by MyroslavKoval1999** [vercel/community#6652] [Help] [answered]
### Summary

So, I bought a domain on Squarespace and managed DNS to Vercel. Currently, we are setting up the "Intercom" help center. For some reason, I can't manage a custom domain; "intercom" is asking me to create a CNAME record, and Vercel immediately provides an error message stating that there is a conflict. So, nothing came to work. 

The support wasn't able to help me and asked me to contact Vercel and ask for help from your side, as they don't know why I can't remove a conflict default record. 

The thing is: they want me to replace a core DNS record, which is locked. I have no idea what a solution can be to manage that. And make a subdomain like help.exampledomain.com

The manual setup they are providing still asks to add CNAME record, a different one, but the result is still a conflict between the records

<img width="546" alt="image" src="https://github.com/vercel/community/assets/50668585/9ae5f638-c721-4378-a415-be07339da74e">

<img width="1256" alt="Screenshot 2024-04-26 at 11 40 33 AM" src="https://github.com/vercel/community/assets/50668585/ff006a94-bb07-49a0-a31c-ff6d96e8de23">

I appreciate any help with the issue

Thank you 

### Steps to Reproduce

You can just make a free trial of Intercom and try to set up a help center with a custom domain for that.

**Comment by amyegan** [comment, marked as answer]
Hey @MyroslavKoval1999. You wouldn't want to replace the existing `*` CNAME record since that would stop the domain working with your existing deployments. Instead, you should use a subdomain for Intercom setup like you mentioned.

If you were using `help.exampledomain.com`, then you would add a new record with **Name** set to `help`, **Type** set to `CNAME`, and **Value** set to `us.intercomhelpcenter.com.` (or whichever value Intercom instructs you to use).

Please give that a try and let me know if you run into any trouble

**Reply by MyroslavKoval1999** [reply]
Yes, it solved the issue, thank you so much!
